adr/0001-community-discussions.adoc

```diff
@@ -0,0 +1,70 @@
+= Setup Quarkus Github Discussions to enable the broader Community
+
+* Status: proposed
+* Date: 2021-06-26
+
+== Context and Problem Statement
+
+Quarkus community is growing and until now we've catered very well for core contributors and initial early consumers. 
+
+We have multiple communication channels: https://github.com/quarkusio/quarkus[issues], https://groups.google.com/g/quarkus-dev?pli=1[quarkus-dev mailing list], https://quarkusio.zulipchat.com[zulip], https://stackoverflow.com/questions/tagged/quarkus[stackoverflow]
+
+Isues are great for bugs/feature work. Mailing list for design conversations between developers, chat for watercooler style discussions and stackoverflow for user questions.
+
+This setup has issues though, some are:
+
+- zulip chat is used for a lot of users questions but none of that is easily searchable/discoverable so it is very synchronous,
+- people reported that they don't feel okey posting on quarkus-dev or zulips as it is seems focused on dev work and not so much about community events, jobs, conferences, etc.
+- its hard to monitor as contributor who wants to help answer/ask questions.
+
+How can we improve this situaton and enable the broader community to more easily ask questions and find answers - without it all be relying on just a few Quarkus core contributors?
+
+== Scenarios (optional)
+
+User wants to locate a answer to a question - zulip chats does not show up in google search; stackoverflow might but might not have an answer.
+
+Contributor want to arrange or attend an event around Quarkus - where do he post/look for info on that ?
+
+Contributor/committer are trying to follow new questions and issues - what does he need to track to do that ? today it is alot and it is all different notification sytems and mainly its a big "firehose" with no good ability to filter out what you don't need.
+
+User looking for a job involving Quarkus - where do he go look ?
+
+
+////
+ as well as a lot of social platform presence https://twitter.com/quarkusio/[twitter], https://www.facebook.com/quarkusio/[facebook], https://www.linkedin.com/groups/13789086/[linkedin], https://www.youtube.com/quarkusio[youtube] and https://www.linkedin.com/groups/13789086/[reddit].
+////
+
+== Considered options
+
+* stackoverflow will not go away - thus for this current proposal it does not include Q/A section.
+* "I can't track more places" is a very valid worry; the advantage of discussions is that it shows up in your normal github notifcations which you can mute, filter,etc. like anything else on Github. Thus it gives you more control rather than just adding more noise.
```

Comment:
```suggestion
* "I can't track more places" is a very valid worry; the advantage of discussions is that it shows up in your normal github notifications which you can mute, filter,etc. like anything else on Github. Thus it gives you more control rather than just adding more noise.
```